packages/node-core/src/indexer/sandbox.ts

```diff
@@ -46,11 +51,65 @@ export class Sandbox extends NodeVM {
         },
       })
     );
+    this.root = option.root;
+
+    const filename = 'dist/index.js';
+    const sourceMapPath = path.join(this.root, filename);
+
+    if (existsSync(sourceMapPath)) {
+      this.sourceMap = this.decodeSourceMap(sourceMapPath);
+    }
   }
 
   async runTimeout<T = unknown>(duration: number): Promise<T> {
     return timeout(this.run(this.script), duration);
   }
+
+  async convertStack(stackTrace: any) {
```

Comment:
```suggestion
  private async convertStack(stackTrace: string): Promise<string> {
```

Reply:
`Error.stack` is of `any` type and I need to retain it for RegEx `matchAll`